app/helpers/application_helper/toolbar/vm_clouds_center.rb

```diff
@@ -122,7 +122,7 @@ class ApplicationHelper::Toolbar::VmCloudsCenter < ApplicationHelper::Toolbar::B
           :url_parms    => "main_div",
           :send_checked => true,
           :confirm      => N_("Initiate Check Compliance of the last known configuration for the selected items?"),
-          :enabled      => false,
+          :enabled      => true,
```

Comment:
Woried about the `:enabled => true` + `:onwhen => "1+"` combination.

Wouldn't that cause onwhen to get ignored in the pre-react implementation?

Reply:
OK, as far as I can tell, `enabled` has absolutely no effect when `onwhen` is present.
So, up to you :)